eva/constants.py

```diff
@@ -17,3 +17,4 @@
 NO_GPU = -1
 UNDEFINED_GROUP_ID = -1
 IFRAMES = "IFRAMES"
+DEFAULT_FUNCTION_EXPRESSION_COST = 100
```

Comment:
In a future PR, we should look at why we have two `constants.py` files.

Reply:
Sure